examples/incremental_learning/helmet_detection/README.md

```diff
@@ -41,15 +41,15 @@ tar -zxvf video.tar.gz
 ```
 
 ### Prepare Image
-this example use the image:  
+This example uses the image:  
 ```
-kubeedge/sedna-example-incremental-learning-helmet-detection:v0.3.0
+kubeedge/sedna-example-incremental-learning-helmet-detection:v0.4.0
 ```
 
 This image is generated by the script [build_images.sh](/examples/build_image.sh), used for creating training, eval and inference worker.  
 
 ### Create Incremental Job
-in this example, `$WORKER_NODE` is a custom node, you can fill it which you actually run.  
+In this example, `$WORKER_NODE` is a custom node, you can fill it which you actually run.  
```

Comment:
In this example, $WORKER_NODE is a custom node, you can fill it which you actually run.
->
In this example, $WORKER_NODE is an environment variable indicating a customized node: you can fill its value with the one you actually run.

Reply:
sorry about this mistake. I have fix the rebase error